Automattic/wp-calypso
# Resolve caching issue: move images to the client assets folder

#### Proposed Changes

* Fixed image caching issue (currently there are wrong/old illustrations in staging and production envs)

I've found the solution following the simple rules described here:
- https://github.com/Automattic/wp-calypso/blob/trunk/static/images/README.md
- https://github.com/Automattic/wp-calypso/blob/trunk/client/assets/images/README.md

#### Testing Instructions

* Go to `/setup/importList?siteSlug={SLUG}&from=`
* Check if there is a new illustration



Related to #64608
